src/soc/ibm/power9/istep_13_scom.h

```diff
@@ -535,36 +663,44 @@
 #define DDRPHY_DP16_WR_VREF_VALUE0_RANK_PAIR2_P0_0	0x8000025E0701103F
 #define DDRPHY_DP16_WR_VREF_VALUE0_RANK_PAIR3_P0_0	0x8000035E0701103F
 #define WR_VREF_RANGE_DRAM0				49
-#define WR_VREF_VALUE_DRAM0				55
+#define WR_VREF_VALUE_DRAM0				50
+#define WR_VREF_VALUE_DRAM0_LEN				6
 #define WR_VREF_RANGE_DRAM1				57
-#define WR_VREF_VALUE_DRAM1				63
+#define WR_VREF_VALUE_DRAM1				58
+#define WR_VREF_VALUE_DRAM1_LEN				6
 
 #define DDRPHY_DP16_WR_VREF_VALUE1_RANK_PAIR0_P0_0	0x8000005F0701103F
 #define DDRPHY_DP16_WR_VREF_VALUE1_RANK_PAIR1_P0_0	0x8000015F0701103F
 #define DDRPHY_DP16_WR_VREF_VALUE1_RANK_PAIR2_P0_0	0x8000025F0701103F
 #define DDRPHY_DP16_WR_VREF_VALUE1_RANK_PAIR3_P0_0	0x8000035F0701103F
 #define WR_VREF_RANGE_DRAM2				49
-#define WR_VREF_VALUE_DRAM2				55
+#define WR_VREF_VALUE_DRAM2				50
+#define WR_VREF_VALUE_DRAM2_LEN				6
 #define WR_VREF_RANGE_DRAM3				57
-#define WR_VREF_VALUE_DRAM3				63
+#define WR_VREF_VALUE_DRAM3				58
+#define WR_VREF_VALUE_DRAM3_LEN				6
 
 #define DDRPHY_DP16_WR_VREF_CONFIG0_P0_0		0x8000006C0701103F
 #define WR_CTR_1D_MODE_SWITCH				48
 #define WR_CTR_RUN_FULL_1D				49
 #define WR_CTR_2D_SMALL_STEP_VAL			52
-#define WR_CTR_2D_BIG_STEP_VAL			56
-#define WR_CTR_NUM_BITS_TO_SKIP			59
-#define WR_CTR_NUM_NO_INC_VREF_COMP			62
+#define WR_CTR_2D_BIG_STEP_VAL				53
+#define WR_CTR_2D_BIG_STEP_VAL_LEN			4
+#define WR_CTR_NUM_BITS_TO_SKIP				57
```

Comment:
```suggestion
#define WR_CTR_NUM_BITS_TO_SKIP				57
#define WR_CTR_NUM_BITS_TO_SKIP_LEN			3
```
Not used, but for completeness